packages/terra-list/docs/guides/MultiSelectList.md

```diff
@@ -90,7 +90,7 @@ Finally we need to check if the item is selected. As we support IE10 & 11, we ca
     );
   }
 ```
-Then we can implement a method to loop through our data and create the list item with our methods and call it from our render method. Making special note to assign the aria role of "listbox" for the list, as it is required for accessibility with selectable list options. Ensuring that we add the appropriate aria label for multiple selections.
+Then we can implement a method to loop through our data and create the list item with our methods and call it from our render method. Making special note to assign the aria role of `"listbox"` for the list, as it is required for accessibility with selectable list options. Ensuring that we add the appropriate aria label for multiple selections, `aria-multiselectable`.
```

Comment:
Can we rephrase the second sentence to say `In addition, we need to assign the aria role`

Reply:
Updated.